.github/package.yml

```diff
@@ -0,0 +1,32 @@
+name: Package
```

Comment:
This should be inside `.github/workflows/`. CMIIW

Reply:
correct. fixed it now.